packages/field/test-suites/InteractionStateMixin.suite.js

```diff
@@ -34,6 +34,7 @@ export function runInteractionStateMixinSuite(customConfig) {
               this.dispatchEvent(
                 new CustomEvent('model-value-changed', { bubbles: true, composed: true }),
```

Comment:
It;s just a test. But without `composed:true` would be the right way now :)

Reply:
done :)